specs/serving/knative-api-specification-1.0.md

```diff
@@ -2215,6 +2215,26 @@ Max: 1
    <td><strong>Schema Requirement</strong>
    </td>
   </tr>
+  <tr>
+   <td><code>readOnlyRootFilesystem</code>
```

Comment:
In https://github.com/knative/specs/pull/73/files#L1523 it is mentioned that "only runAsUser may be specified". This is in contracdictionon to this section.